app/Models/Rule.php

```
@@ -0,0 +1,125 @@
+     * @var array<string, string>
+     */
+    protected $casts = [
+        'stop_processing' => 'boolean',
+        'is_active' => 'boolean',
+        'order' => 'integer',
+    ];
+
+    /**
+     * Get the user that owns the rule.
+     */
+    public function user(): BelongsTo
```

PHPStan (2.1.15)

Method App\Models\Rule::user() return type with generic class Illuminate\Database\Eloquent\Relations\BelongsTo does not specify its types: TRelatedModel, TDeclaringModel


(missingType.generics)